CheatSheets/html-cheatsheet.md

```diff
@@ -39,19 +40,21 @@
 | `<hr/>`                        | In addition to switching to the next line, this tag also drawsa horizontal bar to indicate the end of the section.             |
 | `<strike>...</strike>`         | Another old tag, this is used to draw a line atthe center of the text, so as to make it appearunimportant or no longer useful. |
 | `<cite>...</cite>`             | Tag for citing author of a quote.                                                                                              |
-| `<del>...</del>`               | Pre-formatted, ‘monospace’ text laid out with whitespace inside the element intact.                                            |
-| `<ins> … </ins>`               | Denotes text that has been inserted into the webpage.                                                                          |
-| `<blockquote> … </blockquote>` | Quotes often go into this tag. Is used in tandem with the <cite> tag.                                                          |
+| `<blockquote> … </blockquote>` | Quotes often go into this tag. Is used in tandem with the `<cite>`tag.                                                         |
 | `<q> … </q>`                   | Similar to the above tag, but for shorter quotes.                                                                              |
 | `<abbr> … </abbr>`             | Denotes abbreviations, along with the full forms.                                                                              |
 | `<address> … </address>`       | Tag for specifying author’s contact details.                                                                                   |
 | `<dfn> … </dfn>`               | Tag dedicated for definitions.                                                                                                 |
 | `<code> … </code>`             | This is used to display code snippets within a paragraph.                                                                      |
+
+| `<aside> … </aside>`           | Its a Block level element that defines content aside from the content in which it is Placed.                                   |
+
 | `<sub> … </sub>`               | Used for writing a subscript (smaller font just below the mid-point of normal font).                                           |
 | `<sup> … </sup>`               | Similar to the above tag, but for superscripting.                                                                              |
 | `<small> … </small>`           | Reduces text size. In HTML5, it often refers to redundant or invalid information.                                              | 
 | `<strong> … </strong>`         | This element indicates that its contents have strong importance or urgency. Browsers usualy render the contents in bold type.  |
```

Comment:
Those tags break the table format. Please Fix it.

Reply:
okay